src/lib389/lib389/cli_conf/plugins/rootdn_ac.py

```diff
@@ -20,8 +22,60 @@
 }
 
 
+def validate_args(args):
+    # validate the args
+
+    if args.close_time is not None:
+        try:
+            int(args.close_time)
+            if len(args.open_time) != 4:
+                raise ValueError("The close time must be a 4 digit number: HHMM")
+        except:
+            raise ValueError("The close time must be a 4 digit number: HHMM")
+
+    if args.open_time is not None:
+        try:
+            int(args.open_time)
+            if len(args.open_time) != 4:
+                raise ValueError("The open time must be a 4 digit number: HHMM")
```

Comment:
As we're specifying the format, I'm thinking, maybe we should validate it accordingly? Like HH is 00-23 and MM is 00-59

Reply:
fixed